auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java

```diff
@@ -299,11 +298,11 @@ private void deleteCredentialAndRedirect() {
             return;
         }
 
-        CredentialsApiHelper credentialsApiHelper = CredentialsApiHelper.getInstance(getActivity());
-        credentialsApiHelper.delete(mCredential)
-                .addOnCompleteListener(new OnCompleteListener<Status>() {
+        AuthUI.getInstance()
+                .delete(getActivity())
```

Comment:
This now also deletes the underlying Firebase Auth account in addition to deleting the credentials.  This has two new properties:

- Will fail if there is no currently signed in user
- Will fail if we don't have recent authentication (last 5 minutes)

Did you intend to change this behavior?